src/main/java/Menu.java

```diff
@@ -0,0 +1,38 @@
+import com.github.lalyos.jfiglet.FigletFont;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class Menu {
+    private static final int OPTION_DATA_ENTRY = 1;
+    private static final int OPTION_SCHOOL_MANAGEMENT = 2;
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String welcomeMenu = FigletFont.convertOneLine("Iron-School");
```

Comment:
I'd rather use this approach, so we can avoid the creation of a just single use variable.
System.out.println(FigletFont.convertOneLine("Iron-School"));

Reply:
refactored thx